### PR TITLE
Copypasta from the layouts project created a path issue

### PR DESCRIPTION
**Pull request by bwestergard**
PR @lanetix/engineers

### CODE REVIEW

File: app/templates/lib/as-user-transaction.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict'
-const config = require('layouts/config')
+const config = require('../config')
```
Comment (apechimp) [inline app/templates/lib/as-user-transaction.js]: why not `<%= name %>`?

Comment (bwestergard) [inline app/templates/lib/as-user-transaction.js]: So the microservices generator sets up the symlink for you?

Comment (apechimp) [inline app/templates/lib/as-user-transaction.js]: [indeed!](https://github.com/lanetix/generator-lanetix-microservice/blob/master/app/templates/_package.json#L11)

Comment (bwestergard) [inline app/templates/lib/as-user-transaction.js]: :+1:

**Comment by apechimp** [comment]
:sparkles::sparkles::sparkles::sparkles:
:sparkles:LGTM:sparkles:
:sparkles::sparkles::sparkles::sparkles:

**Pull request merged**